targon/internal/targon/mongo.go

```diff
@@ -18,3 +18,15 @@ func SyncMongo(core *Core, block int) error {
 	_, _ = minerInfoCol.InsertOne(ctx, minerInfo)
 	return nil
 }
+
+func StoreEmissions(core *Core, emissionsData EmissionsData) error {
```

Comment:
this can easily be merged with syncmongo, as both are essentially doing the same thing